### PR TITLE
Socho changes

### PR DESCRIPTION
**Pull request by Socho009**
//

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
```
Comment (assem-ch) [inline manage.py]: Delete the database from git control, its not needed , if you have initial data that should be shared with the project, implement it as [fixtures](https://docs.djangoproject.com/en/2.0/howto/initial-data/)

File: manage.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+import os
+import sys
```
Comment (assem-ch) [inline manage.py]: create a .gitignore file and remove all binary files from git including *.pyc..  you can get prepared gitignores from here: https://github.com/github/gitignore/blob/master/Python.gitignore

File: music/admin.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+
+from .models import Album
+from .models import Song
+
+admin.site.register(Album)
```
Comment (assem-ch) [inline music/admin.py]: I suggest to use this method for Admin models:
```
@admin.register(Album)
class AlbumAdmin(admin.ModelAdmin):
      pass

```

Cause this allows you to customize later

File: music/migrations/0002_song_is_favorite.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+    dependencies = [
+        ('music', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='song',
+            name='is_favorite',
+            field=models.BooleanField(default=False),
```
Comment (assem-ch) [inline music/migrations/0002_song_is_favorite.py]: __pycache__ folder should not be in git, add it to gitignore and remove it from git

File: music/models.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+class Album(models.Model):
+    artist = models.CharField(max_length=250)
+    album_title = models.CharField(max_length=500)
+    genre=models.CharField(max_length=100)
+    album_logo=models.CharField(max_length=1000)
+
+    def get_absolute_url(self):
+        reverse('music:detail',kwargs={'pk':self.pk})
+
+
```
Comment (assem-ch) [inline music/models.py]: one empty line is enough

File: music/models.py [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    def __str__(self):
+        return self.album_title +' , '+ self.artist
+
+class Song(models.Model) :
+    album=models.ForeignKey(Album,on_delete=models.CASCADE)
+    file_type=models.CharField(max_length=10)
+    song_title=models.CharField(max_length=250)
+    is_favorite=models.BooleanField(default=False)
+    def __str__(self):
```
Comment (assem-ch) [inline music/models.py]: you need one empty line here

File: music/models.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+
+    def __str__(self):
+        return self.album_title +' , '+ self.artist
+
+class Song(models.Model) :
+    album=models.ForeignKey(Album,on_delete=models.CASCADE)
+    file_type=models.CharField(max_length=10)
+    song_title=models.CharField(max_length=250)
+    is_favorite=models.BooleanField(default=False)
```
Comment (assem-ch) [inline music/models.py]: Boolean field are by default False so no need to define the default argument unless to set it to `true`

File: website/settings.py [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = ')1jpnl5!2w@h)qytlp#960&y(s3wgr)5(n!iz&ku0q^box-r9%'
```
Comment (assem-ch) [inline website/settings.py]: Secret keys should not published, you need to recover it from environment,  we can discuss this later

File: website/settings.py [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'music.apps.MusicConfig',
```
Comment (assem-ch) [inline website/settings.py]: you can include it as 'music' short

**Review by assem-ch — CHANGES_REQUESTED** [review]

File: manage.py (hunk 1 above)
Comment (Socho009) [reply manage.py]: I put the migrations folder in .gitignore to take the database off git control

Comment (assem-ch) [reply manage.py]: the migrations should be in git , cause it something that you have to share with your collaborators to do the same change you do on database.. only binary database should not be there

Comment (Socho009) [reply manage.py]: Okey I will add it again and remove database binary

File: manage.py (hunk 2 above)
Comment (Socho009) [reply manage.py]: Done

File: music/admin.py (hunk 3 above)
Comment (Socho009) [reply music/admin.py]: Done for both models

File: music/migrations/0002_song_is_favorite.py (hunk 4 above)
Comment (Socho009) [reply music/migrations/0002_song_is_favorite.py]: Done

File: music/models.py (hunk 5 above)
Comment (Socho009) [reply music/models.py]: Done

File: music/models.py (hunk 6 above)
Comment (Socho009) [reply music/models.py]: Done

File: music/models.py (hunk 7 above)
Comment (Socho009) [reply music/models.py]: Done

File: website/settings.py (hunk 8 above)
Comment (Socho009) [reply website/settings.py]: Okey

File: website/settings.py (hunk 9 above)
Comment (Socho009) [reply website/settings.py]: Done

**Pull request merged**